changelogs/unreleased/gh-5843-fiber-wakeup.md

```diff
@@ -0,0 +1,6 @@
+## bugfix/core
+
+Remove assertion raised in case of `fiber_wkeup()` get called with
+dead fibers. Due to backward compatibility we allow such calls for
+release builds but do not in debug builds. In result there was
+inconsistency between program behaviour.
```

Comment:
You missed the ticket reference in form of `(gh-xxxx)`.

Reply:
fixed, thanks